### ISSUE TITLE
bash vi-mode triggers (arg: 200) prompt when pasting

### ISSUE THREAD

**Issue by hackel** [Guake/guake#947]
When pasting, I am thrown into command mode and see an "(arg: 200)" prompt before my pasted text.  The issue is described here:
https://www.reddit.com/r/bash/comments/5pelmy/whats_going_on_with_bracketed_paste_vi_mode/

It appears to be caused by bracketed paste functionality.  I do *not* experience this in gnome-shell so I suspect it may actually be a libvte issue.  If I disable bracketed paste with `printf "\e[?2004l"`, the issue goes away and content pastes normally and stays in insert mode.  Command\\"movement" mode is normally enabled by pressing escape, so apparently bracketed paste is triggering this somehow.

This just started happening for me, I believe with the latest update from Guake 0.8.5 to 0.8.8.  (Ubuntu 17.04)

**Comment by egmontkob** [comment]
I think it's not related to the bracketed paste feature per se, but to a bug in how VTE used to implement this feature. See https://bugs.launchpad.net/ubuntu/+source/vte/+bug/1506166. This has been fixed in VTE's GTK+-3 version for ages; Guake is yet to catch up.

**Comment by jshute96** [comment]
This is coming from readline.  I found that I can fix it by creating an empty ~/.inputrc.  There is something in /etc/inputrc that causes this broken behavior.

**Comment by gsemet** [comment]
can you add a note in the README please?

**Comment by asvc** [comment]
Check this out: https://unix.stackexchange.com/a/125134/57509

**TL;DR**
Disable bracketed paste and use un-bind snippet from the last comment.
Works in st and quake.

Shameless plug:
https://github.com/stoptracking/playbook/blob/master/roles/dotfiles/files/bashrc
https://github.com/stoptracking/playbook/blob/master/roles/dotfiles/files/inputrc

**Comment by egmontkob** [comment]
Let's please not recommend ancient and obsolete workarounds.

This issue should've been closed along with the release of Guake 3.0, about 1.5 years ago. As of Guake 3.0 this bug no longer occurs.

The linked stackexchange answer is 5 years old.

Bracketed paste mode is a nice fix for a security issue: it protects against auto-executing pasted comment which might easily contain harmful commands. Anyone who recommends turning it off actually wishes to open up a security hole for others. It's a truly bad idea.

If anyone still faces the issue reported here, they should upgrade their Guake to version 3.0 or newer. That's it.

**Comment by gsemet** [comment]
Ok, so i close this ticket as fixed. Thanks
